src/app/app.module.ts

```diff
@@ -37,9 +35,9 @@ import { ChatState } from './shared/store/chat.state';
 import { HttpClientModule, HttpClient } from '@angular/common/http';
 import { TranslateModule, TranslateLoader } from '@ngx-translate/core';
 import { TranslateHttpLoader } from '@ngx-translate/http-loader';
+import { DateAdapter } from '@angular/material/core';
```

Comment:
It is recommended to sort the imported files in the following order:

1. angular libs
2. external libraries
3. internal files

You can use [this extension for VSCode](https://marketplace.visualstudio.com/items?itemName=mike-co.import-sorter) to fix the following errors more easily